source/framework/vx_obj_desc.c

```diff
@@ -180,7 +180,10 @@ tivx_obj_desc_t *ownObjDescAlloc(vx_enum ref_type, vx_reference ref)
             /* init entry that is found */
             tmp_obj_desc->obj_desc_id = (uint16_t)idx;
             tmp_obj_desc->scope_obj_desc_id = (vx_enum)TIVX_OBJ_DESC_INVALID;
-            tmp_obj_desc->in_node_done_cnt = 0;
+            for (uint16_t pipeId = 0; pipeId < TIVX_GRAPH_MAX_PIPELINE_DEPTH - 1U; pipeId++)
```

Comment:
Does this style of for loop with the declaration compile for all the targets required? (I seem to remember having to declare the counter outside the loop)

Reply:
Thanks for the hint, I pushed a fix for this file aswell as another error in the test code when compiling for the DSP target.